src/components/organisms/Feedbacks/FeedbackCard.jsx

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components';
+import Card, { variants as cardVariants } from '../../atoms/Card';
+
+const Picture = styled.picture`
+  display: block;
+  width: 4em;
+  height: 4em;
+`;
+
+const Image = styled.img`
+  display: block;
+  width: 100%;
+  height: auto;
+
+  border-radius: 1em;
+`;
+
+const Person = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  margin: -0.5em;
+
+  & > * {
+    margin: 0.5em;
+  }
+`;
+
+const PersonDescription = styled.div``;
+
+const Title = styled.h3`
+  margin: 0 0 0.5em;
+
+  color: ${({ theme }) => theme.colors.text};
+  font-weight: bold;
+`;
+
+const SubTitle = styled.p`
+  margin: 0;
+
+  color: ${({ theme }) => theme.colors.paleText};
+`;
+
+const Wrapper = styled(Card)`
+  break-inside: avoid;
+
+  page-break-inside: avoid;
```

Comment:
I think there is no need for these styles here. You have the same thing in `Content` component